lib/content-services/src/lib/directives/auto-focus.directive.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { AfterContentInit, Directive, ElementRef } from "@angular/core";
+
+@Directive({
+  selector: "[adf-auto-focus]"
+})
+export class AutoFocusDirective implements AfterContentInit {
+  public constructor(private el: ElementRef) {}
+
+  public ngAfterContentInit() {
+    setTimeout(() => {
```

Comment:
I don't quite understand why should we use `setTimeout()`?

Reply:
Yes it should work then as well